app/src/main/java/at/roteskreuz/stopcorona/screens/savedIDs/SavedIDsFragment.kt

```diff
@@ -0,0 +1,62 @@
+package at.roteskreuz.stopcorona.screens.savedIDs
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.appcompat.app.ActionBar
+import androidx.appcompat.widget.Toolbar
+import androidx.fragment.app.Fragment
+import at.roteskreuz.stopcorona.R
+import at.roteskreuz.stopcorona.screens.base.CoronaPortraitBaseActivity
+import at.roteskreuz.stopcorona.skeleton.core.screens.base.activity.startFragmentActivity
+import at.roteskreuz.stopcorona.skeleton.core.screens.base.fragment.BaseFragment
+import kotlinx.android.synthetic.main.saved_ids_fragment.*
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+/**
+ * Screen to inform the user about how he can delete his exposure keys.
+ */
+class SavedIDsFragment : BaseFragment(R.layout.saved_ids_fragment) {
```

Comment:
I find the name confusing, This is not about IDs, it´s about Temporarily Exposure Keys

Your KDoc explains it well.

Suggestion: **InfoOnDeleteTemporarilyExposureKeysFragment**